### PR TITLE
Add attribution metadata

### LINKED ISSUE #48: Update 2018-02-27-pocean-timeSeries-demo to include attribution information
- xref: https://github.com/ioos/notebooks_demos/issues/366

Per a recent discussion concerning attribution, we can update the [pocean timeSeries demo](https://ioos.github.io/ioos_code_lab/content/code_gallery/data_management_notebooks/2018-02-27-pocean-timeSeries-demo.html) to include an attribution step.

From the discussion:

> Organization W collected the oceanographic data data;
> Organization X received the data from Organization W and published it;
> Organization Y downloaded the data from Organization X and QC'd the datasets
> Organization Z took the QC's data from Organization Y and generated netCDF for publication
> Organization G reviewed and uploaded the data to its ERDDAP for wider distribution.
> 
> Question 1: Who is the data creator?
> 
> Question 2: Will acknowledging the chain of custody in the global attribute "Summary" sufficient?

https://github.com/ioos/ioos_code_lab/blob/main/jupyterbook/content/code_gallery/data_management_notebooks/2018-02-27-pocean-timeSeries-demo.ipynb

From @ocefpaf:
For future reference (copied from the ioos_tech thread):

```
creator_name: First Last
creator_email: firstlast@gmail.com
creator_url: http://www.myorg.org/
creator_type: person
creator_institution: My Institution
institution: Usually the sama as the above
publisher_name: RA Data Manager
publisher_email: ra@gmail.com
publisher_url: http://www.ra.org/
publisher_type: position
publisher_institution: The RA
program: List of Programs
project: List of projects
contributor_name: List of names
contributor_role: Principal Investigator, Field Program Manager, Data Manager, Quality Assessment
history: File generated my-awesome-script.py
```
See: https://data.nodc.noaa.gov/thredds/dodsC/ioos/nanoos/ohsucmop/woody/woody.240.A.CTD/200911-926.nc.html

We should modify it to be closer to https://ioos.github.io/ioos-metadata/ioos-metadata-profile-v1-2.html#attribution

### PR DESCRIPTION
**Pull request by ocefpaf**
Closes #48

### CODE REVIEW

File: jupyterbook/content/code_gallery/data_management_notebooks/2017-05-14-running_compliance_checker.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
      "name": "stdout",
```
Comment (MathewBiddle) [inline jupyterbook/content/code_gallery/data_management_notebooks/2017-05-14-running_compliance_checker.ipynb:31]: <p>Looks like this is an error message?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (MathewBiddle) [inline jupyterbook/content/code_gallery/data_management_notebooks/2017-05-14-running_compliance_checker.ipynb:31]: <p>Looks like this is an error message?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (ocefpaf) [reply jupyterbook/content/code_gallery/data_management_notebooks/2017-05-14-running_compliance_checker.ipynb:31]: <p>Argh. This commit should not be here. I based it of the wrong branch. Let me remove it.</p>

Comment (ocefpaf) [reply jupyterbook/content/code_gallery/data_management_notebooks/2017-05-14-running_compliance_checker.ipynb]: I remove that commit it was from the wrong branch.

**Comment by MathewBiddle** [comment]
I'm getting an error with UDUNITS-2:

`OSError: [UT_OPEN_ARG] Failed to open UDUNITS-2 XML unit database: "b'No such file or directory'"`

image
image

**Review by MathewBiddle — COMMENTED** [review]
https://github.com/ioos/ioos_code_lab/pull/63#issuecomment-1040775758

**Comment by MathewBiddle** [comment]
Maybe I need to update `compliance-checker`?

**Comment by MathewBiddle** [comment]
okay, updated compliance-checker with `conda update compliance-checker`

A bunch of stuff updated and now the notebook runs just fine. Thanks for the fix!

**Comment by ocefpaf** [comment]
> okay, updated compliance-checker with `conda update compliance-checker`
> 
> A bunch of stuff updated and now the notebook runs just fine. Thanks for the fix!

BTW, where you on Windows, loading the env on one HDD but running on another (maybe a network one)?
The reason I ask is b/c `udunits2` had that issue but we never nailed it. If a simple update solved it I guess we'll never know what solved it :grimacing:

**Comment by MathewBiddle** [comment]
I am on Windows but don't think I was using two HDDs.

**Pull request merged**